deepchecks/vision/checks/performance/performance_report.py

```diff
@@ -0,0 +1,259 @@
+                    test_scores_dict = dict(zip(test_scores_class['Metric'], test_scores_class['Value']))
+                    train_scores_dict = dict(zip(train_scores_class['Metric'], train_scores_class['Value']))
+                    # Calculate percentage of change from train to test
+                    diff = {score_name: _ratio_of_change_calc(score, test_scores_dict[score_name])
+                            for score_name, score in train_scores_dict.items()}
+                    failed_scores = [k for k, v in diff.items() if v > threshold]
+                    if failed_scores:
+                        for score_name in failed_scores:
+                            explained_failures.append(f'{score_name} for class {class_name} '
+                                                      f'(train={format_number(train_scores_dict[score_name])} '
+                                                      f'test={format_number(test_scores_dict[score_name])})')
+            else:
+                test_scores_dict = dict(zip(test_scores['Metric'], test_scores['Value']))
+                train_scores_dict = dict(zip(train_scores['Metric'], train_scores['Value']))
+                # Calculate percentage of change from train to test
+                diff = {score_name: _ratio_of_change_calc(score, test_scores_dict[score_name])
+                        for score_name, score in train_scores_dict.items()}
+                failed_scores = [k for k, v in diff.items() if v > threshold]
+                if failed_scores:
+                    for score_name in failed_scores:
+                        explained_failures.append(f'{score_name}: '
+                                                  f'train={format_number(train_scores_dict[score_name])}, '
+                                                  f'test={format_number(test_scores_dict[score_name])}')
+            if explained_failures:
+                message = '\n'.join(explained_failures)
+                return ConditionResult(False, message)
+            else:
+                return ConditionResult(True)
+
+        return self.add_condition(f'Train-Test scores relative degradation is not greater than {threshold}',
+                                  condition)
+
+    def add_condition_class_performance_imbalance_ratio_not_greater_than(
+        self: PR,
+        threshold: float = 0.3,
+        score: str = None
+    ) -> PR:
+        """Add condition.
+
+        Verifying that relative ratio difference
+        between highest-class and lowest-class is not greater than 'threshold'.
+
+        Args:
+            threshold: ratio difference threshold
+            score: limit score for condition
+
+        Returns:
+            Self: instance of 'ClassPerformance' or it subtype
+
+        Raises:
+            DeepchecksValueError:
+                if unknown score function name were passed;
+        """
+        if score is None:
+            score = next(iter(MULTICLASS_SCORERS_NON_AVERAGE))
```

Comment:
should probably be next on get_scorers_list. Maybe should save this result in an internal member?